src/cellfinder_core/detect/filters/volume/ball_filter.pyx

```diff
@@ -21,7 +21,8 @@ cdef class BallFilter:
 
     cdef:
         uint THRESHOLD_VALUE, SOMA_CENTRE_VALUE
-        uint ball_xy_size, tile_step_width, tile_step_height
+        uint ball_xy_size, ball_z_size, tile_step_width, tile_step_height
+        uint middle_z_idx
```

Comment:
Does having this in one line go over 80/88 chars?
```suggestion
        uint ball_xy_size, ball_z_size, tile_step_width, tile_step_height, middle_z_idx
```

Hopefully one day black will work https://github.com/psf/black/issues/359